installer/datafiles/base_omsagent.data

```diff
@@ -191,13 +191,33 @@ fi
 #   6. Run the %postun hook of the old package.
 #
 # Thus, if we're an upgrade, skip all of this cleanup
+ETC_DIR=/etc/opt/microsoft/omsagent
+WORKSPACE_REGEX='^[0-9a-fA-F]{8}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{12}$'
 if ${{PERFORMING_UPGRADE_NOT}}; then
-   # Clean up directory tree (created via PostInstall) if dirs are empty
+   # Clean up /var directory tree
    rm -rf /var/opt/microsoft/omsagent 2> /dev/null
-
+   
+   # Clean up /etc/opt directory tree (created via PostInstall) if dirs are empty
+   # If multi-workspace scenario
+   for ws_id in `ls -1 $ETC_DIR | grep -E $WORKSPACE_REGEX`
+    do
+        echo "Deleteting certs and conf directories for workspace $ws_id if empty..."
```

Comment:
nit: misspelling, should be "Deleting certs and conf directories for workspace $ws_id if empty..."